### ISSUE TITLE
listGroupSubheader has incompatible return type to list

### ISSUE THREAD

**Issue by alexanderkiel** [aforemny/material-components-web-elm#57]
```
listGroup []
    [ listGroupSubheader [] [ text "Folders" ]
    , list listConfig
        [ listItem [] [ text "Folder" ]
        , listItem [] [ text "Folder" ]
        ]
    , listGroupSubheader [] [ text "Files" ]
    , list listConfig
        [ listItem [] [ text "File" ]
        , listItem [] [ text "File" ]
        ]
    ]
```

listGroupSubheader returns a ListItem but list return Html, so that code can't work.

**Comment by aforemny** [comment]
Hi @alexanderkiel, thank you for report this. You are right, the type signature of listGroupSubheader does not make any sense as it is. Unfortunately, this will be a major update and I want to get a few other features released before a 3.0.0 release.

As a work-around you should be able to write your own `listGroupSubheader` like this:

```
listGroup []
    [ Html.span [ Html.Attributes.class "mdc-list-group__subheader" ] [ text "Folder" ]
    , list listConfig …
    , …
    ]
```

**Comment by aforemny** [comment]
Fixed in 4a25c103007798f8da482a31a32f053a68de6f57. Thank you for your patience!
